Doc/reference/compound_stmts.rst

```diff
@@ -611,11 +611,10 @@ the form "``-> expression``" after the parameter list.  These annotations can be
 any valid Python expression.  The presence of annotations does not change the
 semantics of a function.  The annotation values are available as values of
 a dictionary keyed by the parameters' names in the :attr:`__annotations__`
-attribute of the function object.  If the ``annotations`` import from
-:mod:`__future__` is used, annotations are preserved as strings at runtime which
-enables postponed evaluation.  Otherwise, they are evaluated when the function
-definition is executed.  In this case annotations may be evaluated in
-a different order than they appear in the source code.
+attribute of the function object. Used annnotations are preserved as strings at
+runtime which enables postponed evaluation. In this case annotations may be
+evaluated in In this case annotations may be evaluated in a different order
+than they appear in the source code.
```

Comment:
```suggestion
evaluated in a different order than they appear in the source code.
```

Reply:
Thanks for the contribution but please avoid reviewing this PR, unless it is marked so. I'm trying to have a clean commit log and little documentation suggestions like this will mess with that log.